### ISSUE TITLE
generate.sh error

### ISSUE THREAD

**Issue by IconicKestrel** [7thFACTOR/horus_ui#26]
![2024-03-08_09-59](https://github.com/7thFACTOR/horus_ui/assets/126169462/155d88a7-fdaa-48ec-9138-71633e536b03)

Trying to build on arch linux, after running sh ./setup.sh, when running sh ./generate.sh I am seeing this error

**Comment by nekitu** [comment]
yes, sorry, main branch is really old, not maintained, was wip anyway.
I am currently doing work in dev branch, but until I will not finish docking and other api changes, I will not merge it into main.

**Comment by nekitu** [comment]
Linux was just experimental, also I am using CMake now in the dev branch
